trade-finance-manager-ui/server/routes/index.test.js

```diff
@@ -19,14 +20,15 @@ describe('routes index', () => {
   });
 
   it('should setup all routes', () => {
-    expect(use).toHaveBeenCalledTimes(8);
+    expect(use).toHaveBeenCalledTimes(9);
     expect(use).toHaveBeenCalledWith('/', loginRoutes);
     expect(use).toHaveBeenCalledWith('/case', validateUser, caseRoutes);
     expect(use).toHaveBeenCalledWith('/deals', validateUser, dealsRoutes);
     expect(use).toHaveBeenCalledWith('/facilities', validateUser, facilitiesRoutes);
     expect(use).toHaveBeenCalledWith('/feedback', feedbackRoutes);
     expect(use).toHaveBeenCalledWith('/thank-you-feedback', feedbackThankYouRoutes);
     expect(use).toHaveBeenCalledWith('/user', userRoutes);
+    expect(use).toHaveBeenCalledWith('/utilisation-reports', validateUser, utilisationReportsRoutes);
     expect(use).toHaveBeenCalledWith('/', footerRoutes);
   });
 });
```

Comment:
I really dislike these test files as they feel redundant, but I suppose we can't really remove them at the moment